static/sass/_pattern_card.scss

```
@@ -336,3 +337,39 @@
     }
   }
 }
+
+// XXX: Ant - 01/02/2023
+// There is a new pattern emerging across this site and has been proposed
+// upstream here. Once addressed this local styling can be removed:
+// https://github.com/canonical/vanilla-framework/issues/4640
+@mixin ubuntu-tiered-cards {
+  %tiered-padding {
+    padding: $sp-large;
```

The values here should be .5rem 1rem 4rem, so maybe something like this?

```suggestion
    @extend %section-padding--default;
    padding: $spv--small $sph-large;
```